README.md

```diff
@@ -299,6 +316,36 @@ Since npm does it, we should do it:
 - Scripts like this `"eslint": "eslint"` used to be common to make dependencies executable with `npm run`. Since [`npx`](https://www.npmjs.com/package/npx) (and with yarn of course) this is obsolete. There's a lint rule in the CLI for this.
 - scripts that start with `pre*` or `post*` should only be used as hooks for built-in commands or other scripts
 
+## scriptlint as a module dependency
+
+You can install scriptlint as a dependency and call it as a function like this:
+
+```
+const scriptlint = require("scriptlint");
+
+const scriptlintIssues = scriptlint({
+	strict: true,
+	packageFile: "/Users/foobar/project-dir"
+});
+
+```
+The function returns found issues as an array:
+
+```
+[
+  {
+    "message": "must contain a \"start\" script (mandatory-start)",
+    "type": "warning",
+    "affected": false // script name or false for object rules
+  },
+  {
+    "message": "Use of unix double ampersand (&&) in script 'test' is not allowed, consider using npm-run-all/run-s (no-unix-double-ampersand)",
+    "type": "warning",
+    "affected": "test"
+  }
+]
+```
```

Comment:
How does the success case look like? E.g. no errors. Is the array simply empty?

Reply:
Exactly.